diff --git a/deployment.pri b/deployment.pri
index 57bdd3a..9c5be8f 100644
--- a/deployment.pri
+++ b/deployment.pri
@@ -3,7 +3,7 @@ unix:!android {
         qnx {
             target.path = /tmp/$${TARGET}/bin
         } else {
-            target.path = /usr/bin
+            target.path = /app/bin
         }
         export(target.path)
     }
diff --git a/yubioath-desktop.pro b/yubioath-desktop.pro
index 551416a..9a1c271 100644
--- a/yubioath-desktop.pro
+++ b/yubioath-desktop.pro
@@ -51,7 +51,7 @@ macx {
     pip.commands = python3 -m venv pymodules && source pymodules/bin/activate && pip3 install -r requirements.txt && deactivate
 }
 !macx {
-    pip.commands = pip3 install -r requirements.txt --target pymodules
+    pip.commands = pip3 install --no-deps --prefix $$PREFIX --target pymodules
 }
 
 # Default rules for deployment.
